README.md

```diff
@@ -119,3 +122,22 @@ It is possible to install the proprietary packages via OPKG by doing the followi
 > **Note**:
 > The read-write filesystem is (currently) not large enough to contain all the above packages. It is possible to resize
 > the UBI volumes. However, that must be done with great care.
+
+## Restoring the Official Image
+
+Unless the above instructions have been performed multiple times, reverting back to the official image is as simple as
+booting the previously used boot slot (from Linux):
+```bash
```

Comment:
Not 100% sure if better, but I think that giving the U-Boot commands might be more reliable. But doing it from Linux might be easier, because SSH is possible.

Reply:
The U-Boot instructions are already above. I find the Linux instructions helpful too.